### DISCUSSION TITLE
Assertion Error expected error to have been called exactly '0 times', but it was called once     error('ERROR', [Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'.]) at unknown

### DISCUSSION THREAD

**Discussion by Greshma-CK** [cypress-io/cypress#20010] [Questions and Help]
Hi, I am getting below given error on executing cypress test case.Application is getting launched but getting below given issue on performing any action. Currently i am trying to migrate app from Protractor to cypress. Kindly help with any suggestion or solutions to resolve this issue 

AssertionError
expected error to have been called exactly '0 times', but it was called once
    error('ERROR', [Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'.]) at unknown
node_modules/cypress-fail-on-console-error/dist/index.js:51:1
  49 |         var spy = (0, exports.getIncludedSpy)(spies, config);
  50 |         if (spy) {
> 51 |             chai.expect(spy).to.have.callCount(0);
     | ^
  52 |         }
  53 |     });
  54 | }

View stack trace
 Print to console
    at $Cypress.eval (node_modules/cypress-fail-on-console-error/dist/index.js:51:1)
From previous event:
    at next (http://localhost:4200/__cypress/runner/cypress_runner.js:151297:35)
    at <unknown> (http://localhost:4200/__cypress/runner/cypress_runner.js:151325:17)

**Comment by jjhelguero** [comment]
Whatever you were spying, was called once by your application. You may have to dig deep to figure out why.
